File: week02/day02/draw-pyramid.py
# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was

print("add a number:")
x = int(input())
for i in range(x)  :
    i = ((x-i) * " " + (i * "*") * 2 + "* ")
    print(i)

File: week07/day02/draw-pyramid.js
// Write a program that draws a
// pyramid like this:
//
//
//    *
//   ***
//  *****
// *******
//
// The pyramid should have as many lines as lineCount is

'use strict';

var lineCount = 4;
var star = '';

for (var i = 0; i <= lineCount; i += 1) {

  star += '**';
  console.log(star);
}
